backend/api/Services/RobotService.cs

```diff
@@ -28,5 +40,32 @@ public async Task<IEnumerable<Robot>> ReadAll()
         {
             return await _context.Robots.FirstOrDefaultAsync(robot => robot.Id.Equals(id, StringComparison.Ordinal));
         }
+
+        public async Task<Report> StartMission(Robot robot, string missionId)
+        {
+            var response = await _isarService.StartMission(missionId);
+            if (!response.IsSuccessStatusCode)
+            {
+                throw new MissionException($"Could not start mission with id: {missionId}");
+            }
+            if (response.Content != null)
+            {
+                string? responseContent = await response.Content.ReadAsStringAsync();
+                var isarResponse = JsonSerializer.Deserialize<IsarStartMissionResponse>(responseContent);
+                var report = new Report
+                {
+                    EchoMissionId = "1",
+                    IsarMissionId = isarResponse?.MissionId,
+                    StartTime = DateTimeOffset.UtcNow,
+                    EndTime = DateTimeOffset.UtcNow,
+                    ReportStatus = ReportStatus.InProgress,
+                    Robot = robot,
+                    Log = "log"
+                };
+                _logger.LogInformation($"Mission was started on robot");
```

Comment:
```suggestion
                _logger.LogInformation($"Mission started on robot");
```
Consider logging mission and robot id as well? For better readability of the log

Reply:
It is preferred that logging does not change. Ref
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2254
So I will not add robot id and mission id

Reply:
Interesting. So there should be no logging of situational variables? For example it would be very useful to know in the log which mission failed in case it was a deserialization error etc.

Reply:
The logging message template should not change, but the logging message template can include argument placeholders as I understand it: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-6.0#lmt

Or maybe I have not understood it correctly?

If it does not break with good practice, i think we should in general include the ids when logging about missions, events or robots.

Reply:
Logging to be taken care of by:
https://github.com/equinor/flotilla/issues/199